ProCamp_Task_5_STM32F103_led_blink/Core/Src/main.c

```diff
@@ -93,6 +93,8 @@ int main(void)
   /* USER CODE BEGIN WHILE */
   while (1)
   {
+	  HAL_GPIO_TogglePin(LED1_GPIO_Port, LED1_Pin);
+	  HAL_Delay(200);
```

Comment:
implementation is fine but I will reduce 1 point for using stm32cube and 1 point for using HAL. if it is fine for you - than you can merge. Also 200ms is quite small time for blinking... have you test it with a real LED?